core/platform/email/gae_email_services.py

```diff
@@ -22,29 +22,6 @@
 from google.appengine.api import app_identity
 from google.appengine.api import mail
 
-
-def send_mail_to_admin(subject, body):
-    """Enqueues a 'send email' request with the GAE mail service.
-
-    Args:
-      - subject: str. The subject line of the email.
-      - body: str. The plaintext body of the email.
-    """
-    if feconf.CAN_SEND_EMAILS_TO_ADMIN:
-        if not mail.is_email_valid(feconf.ADMIN_EMAIL_ADDRESS):
-            raise Exception(
-                'Malformed email address: %s' %
-                feconf.ADMIN_EMAIL_ADDRESS)
-
-        app_id = app_identity.get_application_id()
```

Comment:
Yep, either the domain or controller layer is fine, I think.

Could you also add an app_identity service to the platform layer, since it's GAE specific, and reference that, instead of importing google.appengine.api directly in the controller/domain layer?